api/src/pcapi/admin/custom_views/custom_reimbursement_rule_view.py

```diff
@@ -188,6 +188,7 @@ def format_venue(
 ) -> str:
     if model.offererId:
         return ""
+    assert model.offer  # helps mypy when accessing venue below
```

Comment:
On est sûr de toujours avoir `model.offer` ? 
_Au cas où_ est ce qu'on ne préfère pas avoir un truc explicite comme 
```suggestion
    if not model.offer:
        raise SomeThing
```

Reply:
Oui, on est sûr. Il y a une contrainte en base de données qui dit qu'une `CustomReimbursementRule` est liée soit à une offre, soi à une structure : 

https://github.com/pass-culture/pass-culture-main/blob/4634fff0a297cac1f6bbfff389e40f1eed0f2b5f/api/src/pcapi/core/finance/models.py#L410